Makefile

```diff
@@ -19,7 +19,7 @@ clean: ## Clean up prior to building
 setup-dev: SHELL:=/bin/bash
 setup-dev: ## Setup development environment
 	@. utils/install_dependencies.sh
-
+	@poetry run python utils/install_gcs_connector.py
```

Comment:
Hi @Daniel-Considine, I'm qiute concerned that this connector. I feel all our components are expected to either run on dataproc, where the connection to GCP buckets are managed at a deeper level or locally (mostly for debugging or prototyping), where a smaller chunk or sample of the input data is sufficient. 

I feel, having the logic in our repo to magange the gcs_connector.py is quite a burden and a bit out of scope.

Reply:
No worries - I thought it might be useful for testing smaller steps on the stored data without having to download to local/rewrite paths.